dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/SaveContentActionlet.java

```diff
@@ -0,0 +1,87 @@
+package com.dotmarketing.portlets.workflows.actionlet;
+
+import com.dotcms.business.WrapInTransaction;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.portlets.contentlet.business.ContentletAPI;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.workflows.model.*;
+import com.dotmarketing.util.Logger;
+import com.google.common.annotations.VisibleForTesting;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Do the save for a content (checkin)
+ * @author jsanca
+ */
+public class SaveContentActionlet extends WorkFlowActionlet {
+
+	private final ContentletAPI contentletAPI;
+
+	public SaveContentActionlet () {
+
+		this(APILocator.getContentletAPI());
+	}
+
+	@VisibleForTesting
+	protected SaveContentActionlet (final ContentletAPI contentletAPI) {
+
+		this.contentletAPI = contentletAPI;
+	}
+
+	/**
+	 * 
+	 */
+	private static final long serialVersionUID = 1L;
+
+	public String getName() {
+		return "Save content";
+	}
+
+	public String getHowTo() {
+
+		return "This actionlet will checkin the content.";
+	}
+
+	@WrapInTransaction
+	public void executeAction(final WorkflowProcessor processor,
+							  final Map<String, WorkflowActionClassParameter> params) throws WorkflowActionFailureException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.dotmarketing.portlets.workflows.model.WorkflowActionFailureException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)